pytest_perf/runner.py

```diff
@@ -78,7 +79,18 @@ def __init__(self, extras=(), deps=(), control=None):
         spec = f'[{",".join(extras)}]' if extras else ''
         self.stack = contextlib.ExitStack()
         self.control_env = self._setup_env(upstream_url(spec, control), *deps)
-        self.experiment_env = self._setup_env(f'.{spec}', *deps)
+        target = os.environ.get("PYTEST_PERF_WHEEL_TARGET")
+        if target:
+            if spec:
+                # Assuming wheel name is normalised we can derive the package name:
+                name, _, _ = os.path.basename(target).partition("-")
+                # Normalise file URL for both Windows and POSIX:
+                file_url = f"file:///{os.path.abspath(target).replace(os.sep, '/')}"
+                file_url = file_url.replace("file:////", "file:///")
```

Comment:
I'd like to see this functionality extracted as a function rather than inline here.